src/DocAssembler/DocAssembler.Test/DocAssembler.Test.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
all up, why not directly .NET 9 for all projects?

Reply:
Because .NET 8 is LTS. Even versions are LTS, Uneven are shorter lived (for some reason).